src/main/kotlin/nl/info/client/brp/BrpClientService.kt

```diff
@@ -104,18 +106,25 @@ class BrpClientService @Inject constructor(
         auditEvent: String,
         defaultPurpose: String?,
         extractPurpose: (ZaakafhandelParameters) -> String?
-    ): String? = Regex("""ZAAK-\d{4}-\d+""")
-        .find(auditEvent)
-        ?.value
-        ?.let { zaakIdentificatie ->
-            runCatching {
-                val zaak = zrcClientService.readZaakByID(zaakIdentificatie)
-                val parameters = zaakafhandelParameterService.readZaakafhandelParameters(
-                    zaak.zaaktype.extractUuid()
-                )
-                extractPurpose(parameters)
-            }.getOrNull()
-        } ?: defaultPurpose
+    ): String? {
+        LOG.fine("Resolving BRP purpose using auditEvent='$auditEvent', defaultPurpose='$defaultPurpose'.")
```

Comment:
What is the purpose of these log lines? This log level will never be used on real environments. It will not help troubleshooting.

Reply:
If you want to log this then you need to change the log level _for this class only_ in WildFly. You can do this statically in `configure-wildfly.cli`. Maybe you can also do this dynamically (runtime) but I don't know how.